docs/_config.yml

```diff
@@ -59,4 +59,4 @@ autoprefixer:
 
 remote_theme: 18F/uswds-jekyll
 
-theme: jekyll-theme-cayman
\ No newline at end of file
+#theme: jekyll-theme-cayman
```

Comment:
Fixes error:

``
         1: from /usr/local/lib/ruby/gems/2.7.0/gems/jekyll-4.2.0/lib/jekyll/theme.rb:79:in `gemspec'
/usr/local/lib/ruby/gems/2.7.0/gems/jekyll-4.2.0/lib/jekyll/theme.rb:82:in `rescue in gemspec': The jekyll-theme-cayman theme could not be found. (Jekyll::Errors::MissingDependencyException)
``